lib/component_reader.js

```diff
@@ -68,15 +68,17 @@ ComponentReader.prototype.loadTriggerOrAction = function loadTriggerOrAction(nam
         result = require(modulePath);
         return Q.resolve(result);
     } catch (err) {
-        var message = "Trigger or action '%s' is found, but can not be loaded. "
-            + "Please check if the file '%s' is correct.";
+        var message;
 
         if (err.code === 'MODULE_NOT_FOUND') {
-            message = "Trigger or action '%s' is not found. "
-                + "Please check if the path you specified in component.json ('%s') is valid.";
+            message = 'Failed to load file \'%s\': %s';
+            err.message = util.format(message, filename, err.message);
+        } else {
+            message = 'Trigger or action \'%s\' is found, but can not be loaded. '
+                + 'Please check if the file \'%s\' is correct.';
+            err.message = util.format(message, name, filename);
```

Comment:
How about to use a template string 
```js 
err.message = `Trigger or action '${name}' is found...` 
```  
instead of 
```js
message = 'Trigger or action \'%s\' is found...'
err.message = util.format(message, name);
```?